packages/stack/src/navigators/createStackNavigator.tsx

```diff
@@ -17,11 +19,11 @@ import {
   StackNavigationEventMap,
 } from '../types';
 
-type Props = DefaultNavigatorOptions<StackNavigationOptions> &
+export type Props = DefaultNavigatorOptions<StackNavigationOptions> &
```

Comment:
I think this shouldn't be exported. You should use `React.ComponentProps<typeof Stack.Navigator>` or `React.ComponentProps<TypedStackNavigator<ParamList>['Navigator']>` in this case. Otherwise it lacks type information for params.